doc/classes/ResourceImporterDynamicFont.xml

```diff
@@ -1,43 +1,77 @@
+			[b]Grayscale:[/b] Use grayscale antialiasing. This is the approach used by the operating system on macOS, Android and iOS.
+			[b]LCD Subpixel:[/b] Use antialiasing with subpixel patterns to make fonts sharper on LCD displays. This is the approach used by the operating system on Windows and most Linux distributions. The downside is that this can introduce "fringing" on edges, especially on display technologies that don't use standard RGB subpixels (such as OLED displays). The LCD subpixel layout is globally controlled by [member ProjectSettings.gui/theme/lcd_subpixel_layout], which also allows falling back to grayscale antialiasing.
 		</member>
 		<member name="compress" type="bool" setter="" getter="" default="true">
+			If [code]true[/code], uses lossless compression for the resulting font.
 		</member>
 		<member name="fallbacks" type="Array" setter="" getter="" default="[]">
+			List of font fallbacks to use if a glyph isn't found in this dynamic font. Fonts are the beginning of the array are attempted first. See also [member allow_system_fallback].
 		</member>
 		<member name="force_autohinter" type="bool" setter="" getter="" default="false">
+			If [code]true[/code], forces generation of hinting data for the font using [url=https://freetype.org/]FreeType[/url]'s autohinter. This will make [member hinting] effective with fonts that don't include hinting data.
 		</member>
 		<member name="generate_mipmaps" type="bool" setter="" getter="" default="false">
+			If [code]true[/code], this font will have mipmaps generated. This prevents text from looking grainy when a [Control] is scaled down, or when a [Label3D] is viewed from a long distance (if [member Label3D.texture_filter] is set to a mode that displays mipmaps).
+			Enabling [member generate_mipmaps] increases font generation time and memory usage. Only enable this setting if you actually need it.
 		</member>
 		<member name="hinting" type="int" setter="" getter="" default="1">
+			The hinting mode to use. This controls how aggressively glyph edges should be snapped to pixels when rasterizing the font. Depending on personal preference, you may prefer using one hinting mode over the other. Hinting modes other than [b]None[/b] are only effective if the font contains hinting data (see [member force_autohinter]).
+			[b]None:[/b] Smoothest appearance, which can make the font look blurry at small sizes.
+			[b]Light:[/b] Sharp result by snapping glyph edges to pixels on the Y axis only.
+			[b]Full:[/b] Sharpest by snapping glyph edges to pixels on both X and Y axes.
 		</member>
 		<member name="language_support" type="Dictionary" setter="" getter="" default="{}">
+			The list of languages supported by this font. See also [member script_support].
```

Comment:
Probably should be specified that it is for overriding font metadata. And usually it's not necessary to set.